### PR TITLE
update F1 to F3

### PR DESCRIPTION
**Pull request by MaratMussabekov**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/4671
Office 365 F1 was renamed into Office 365 F3 - https://docs.microsoft.com/en-us/office365/servicedescriptions/office-365-platform-service-description/office-365-plan-options
Microsoft 365 F1 was renamed into Microsoft 365 F3 - https://www.microsoft.com/en-us/microsoft-365/compare-microsoft-365-enterprise-plans
Thank you

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9,
kindly do the copy editor review for this PR.

Dear @scanum, for your information

Thank you

File: Teams/business-voice/whats-business-voice.md [hunk 1]
```diff
@@ -32,8 +32,8 @@ Business Voice is right for you if your organization has 300 or fewer people and
 - Microsoft 365 Business Premium
 - Microsoft Apps for business 
 - Microsoft Apps for enterprise
-- Microsoft 365 F1
-- Office 365 F1
+- Microsoft 365 F3
+- Office 365 F3
 - Microsoft and Office 365 Enterprise E1, E3 or E5
```
Comment (JohanFreelancer9) [inline Teams/business-voice/whats-business-voice.md:37]:
```suggestion
- Microsoft and Office 365 Enterprise E1, E3, or E5
```

**Comment by scanum** [comment]
@dstrome PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by MaratMussabekov** [comment]
Fixed by https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/commit/3ac32fbb70cf2042e9976c2a973a2feaee458d85#diff-40581e7a4083387551a935e06f8c0646 and https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/commit/117ebf4ba60ad7b5f42d0bc19a3b3dfa81483168#diff-40581e7a4083387551a935e06f8c0646
Thank you

**Pull request closed without merging**